app/service/news_api_service.rb

```diff
@@ -0,0 +1,16 @@
+class NewsApiService 
+  def self.search_articles(keyword)
+    response = conn.get("/v2/everything??") do |req|
+      req.params["q"] = "#{keyword}"
```

Comment:
Out of curiosity, why do you interpolate keyword? as it is received in the argument you can just set the param equal to _keyword_, unless there is another reason for the interpolation.